src/components/NotificationBox.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { CSSProperties } from 'react';
+import { suomifiTheme, Icon } from 'suomifi-ui-components';
+import { Text } from 'components/ResponsiveComponents';
+import { NamespacesConsumer } from 'react-i18next';
+
+interface Props {
+  style?: CSSProperties;
+  notificationText?: string;
+}
+
+const NotificationBox = ({ style, notificationText }: Props): JSX.Element => (
+  <NamespacesConsumer>
+    {t => (
+      <div
+        style={{
+          display: 'flex',
+          justifyContent: 'center',
+          alignItems: 'center',
+          flexWrap: 'wrap',
+          padding: suomifiTheme.spacing.l,
+          margin: `${suomifiTheme.spacing.m} 0`,
+          ...style,
+          background: suomifiTheme.colors.accentSecondaryLight40,
+          fontSize: '24px',
+        }}
+      >
+        <Icon icon="error" color={suomifiTheme.colors.accentSecondary} />
+        <Text.bold
+          style={{
+            marginLeft: suomifiTheme.spacing.m,
+            verticalAlign: 'middle',
+            fontSize: suomifiTheme.typography.fontSize.body,
```

Comment:
Is it possible that this could conflict with the above static 24px value?

Reply:
The text element seems to get the body text default value so it isn't affected. This was just my observation so probably a good idea to test that a bit more thoroughly.